pkg/virtctl/imageupload/imageupload.go

```diff
@@ -926,3 +935,43 @@ func handleEventErrors(client kubecli.KubevirtClient, pvcName, dvName, namespace
 
 	return nil
 }
+
+func createDataSource(client kubecli.KubevirtClient, name, namespace string) error {
+	labels := make(map[string]string)
+	setDefaultInstancetypeLabels(labels)
```

Comment:
nit - <insert git blame meme here> - `setDefaultInstancetypeLabels` is ugly and should really take the `ObjectMeta` of an object as the only argument as we do in `applyPreferenceAnnotations` etc within `pkg/instancetype`. I can clean this up in a follow up if you don't want to address it as part of this PR.

Reply:
Yes I agree. I'd be happy to refactor it in a seperate PR myself.